code/modules/keybindings/readme.md

```diff
@@ -0,0 +1,40 @@
+# In-code keypress handling system
```

Comment:
This stuff seems inaccurate.

Reply:
The entire readme is inaccurate. It copied from tgstation or TauCeti, and I have no language knowledge to rewrite it if needed.